core/iwasm/common/wasm_application.c

```diff
@@ -600,7 +600,7 @@ execute_func(WASMModuleInstanceCommon *module_inst, const char *name,
             {
 #if UINTPTR_MAX == UINT32_MAX
                 if (argv1[k] != 0 && argv1[k] != (uint32)-1)
-                    os_printf("0x%" PRIxPTR ":ref.extern", (void *)argv1[k]);
+                    os_printf("%p:ref.extern", (void *)argv1[k]);
```

Comment:
iirc, how `%p` converts is implementation-defined.
isn't it a problem for repl?